tasks/ceph_manager.py

```diff
@@ -727,11 +731,11 @@ def do_dump_ops(self):
             for osd in self.live_osds:
                 # Ignore errors because live_osds is in flux
                 self.ceph_manager.osd_admin_socket(osd, command=['dump_ops_in_flight'],
-                                     check_status=False, timeout=30)
+                                     check_status=False, timeout=30, stdout=DEVNULL)
```

Comment:
if we don't want `do_dump_ops()`, why not just disable it using the  `dump_ops_enable` option?

and the whole point of `do_dump_ops()` is for dumping the ops, if we simply redirect the stdout to `/dev/null`, we render this method useless.